helloworld-shell/tests.cloudbuild.yaml

```
@@ -1,5 +1,20 @@
 steps:
 
+- id: 'Build Cop'
+  name: golang
+  entrypoint: /bin/bash
+  args:
+    - '-c'
+    - | # Run build cop only on nightly builds, skip PR's 
+      pr_var=$_PR_NUMBER 
+      if [[ $pr_var ]]
+      then 
+      echo "Skipping Build Cop"
+      else 
+      go get github.com/googleapis/repo-automation-bots/packages/buildcop
+      buildcop -repo=GoogleCloudPlatform/cloud-run-samples -commit_hash=$COMMIT_SHA
+      fi
```

Typical to indent if blocks. Complex enough that stopping on first error is useful. Reducing the mystery of failures by outputting complex commands.

```suggestion
    - | # Run build cop only on nightly builds, skip PR's 
      set -e
      pr_var=$_PR_NUMBER 
      if [[ $pr_var ]]
      then 
        echo "Skipping Build Cop"
      else
        set -x 
        go get github.com/googleapis/repo-automation-bots/packages/buildcop
        buildcop -repo=GoogleCloudPlatform/cloud-run-samples -commit_hash=$COMMIT_SHA
        set +x
      fi
```